lib/desired-caps.js

```diff
@@ -110,6 +110,17 @@ validator.validators.deprecated = function (value, options, key) {
   }
   return null;
 };
+validator.validators.inclusionCaseInsensitive = function (value, options) {
+  if (typeof value !== 'string') {
+    return `unrecognised`;
```

Comment:
Ah. That is a good output. 

As for the strings, the rest should be changed rather than another be added in the wrong format.